proto/sentry_protos/snuba/v1/trace_item_filter.proto

```diff
@@ -0,0 +1,50 @@
+syntax = "proto3";
+
+package sentry_protos.snuba.v1;
+
+import "sentry_protos/snuba/v1/trace_item_attribute.proto";
+
+message AndFilter {
+  repeated TraceItemFilter filters = 1;
+}
+
+message OrFilter {
+  repeated TraceItemFilter filters = 1;
+}
+
+message NotFilter {
+  repeated TraceItemFilter filters = 1;
+}
+
+message ComparisonFilter {
+  enum Op {
+    OP_UNSPECIFIED = 0;
+    OP_LESS_THAN = 1;
+    OP_GREATER_THAN = 2;
+    OP_LESS_THAN_OR_EQUALS = 3;
+    OP_GREATER_THAN_OR_EQUALS = 4;
+    OP_EQUALS = 5;
+    OP_NOT_EQUALS = 6;
+    OP_LIKE = 7; //string only
+    OP_NOT_LIKE = 8; //string only
```

Comment:
can we add `OP_IN` and `OP_NOT_IN?`

Reply:
Done

Reply:
we'd need a new class for this, right? since `value` here isn't a list